setup.py

```diff
@@ -115,30 +116,40 @@ def user_dir():
 
 class DevelopCmd(develop):
     prefix_targets = [
-        ("nbconvert/templates", 'jdaviz-default'),
-        ("voila/templates", 'jdaviz-default')
+        (os.path.join("nbconvert", "templates"), 'jdaviz-default'),
+        (os.path.join("voila", "templates"), 'jdaviz-default')
     ]
 
     def run(self):
         target_dir = os.path.join(sys.prefix, 'share', 'jupyter')
         if '--user' in sys.prefix:  # TODO: is there a better way to find out?
             target_dir = user_dir()
         target_dir = os.path.join(target_dir)
+        is_win = sys.platform.startswith('win')
 
         for prefix_target, name in self.prefix_targets:
             source = os.path.join('share', 'jupyter', prefix_target, name)
             target = os.path.join(target_dir, prefix_target, name)
             target_subdir = os.path.dirname(target)
             if not os.path.exists(target_subdir):
                 os.makedirs(target_subdir)
-            rel_source = os.path.relpath(os.path.abspath(
-                source), os.path.abspath(target_subdir))
+            if not is_win:
+                rel_source = os.path.relpath(os.path.abspath(
+                    source), os.path.abspath(target_subdir))
+            else:  # relpath does not work if source/target on different Windows disks
+                rel_source = os.path.abspath(source)
             try:
-                os.remove(target)
+                if not is_win:
+                    os.remove(target)
+                else:
+                    shutil.rmtree(target)
             except Exception:
                 pass
             print(rel_source, '->', target)
-            os.symlink(rel_source, target)
+            if not is_win:
+                os.symlink(rel_source, target)
+            else:  # Cannot symlink without relpath or Windows admin priv in some OS versions
+                shutil.copytree(rel_source, target)
```

Comment:
Why not try the symlink in case the user DOES have admin rights? 

```suggestion
            try:
                rel_source.symlink_to(target)
            except:  # Cannot symlink without relpath or Windows admin priv in some OS versions
                shutil.copytree(rel_source, target)
```

Also I always forget the direction of the symlink. Did I get the direction right here?